### PR TITLE
Fix eslint errors and run eslint in travis 

### PR DESCRIPTION
**Pull request by cooperq**
cc @ghostwords

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -208,8 +208,8 @@ Badger.prototype = {
                                   function(subdomain){
                                     return subdomain.endsWith(domain);
                                   });
-        _.each(rmvdSubdomains, function(domain){
-          self.storage.setupHeuristicAction(domain, constants.BLOCK);
+        _.each(rmvdSubdomains, function(subDomain){
```
Comment (ghostwords) [inline src/background.js]: Can the above `TODO` now be removed?

Comment (ghostwords) [inline src/background.js]: Oh yes, pretty sure it can; the bug was fixed in 43c1c63aee9fb73a2d69518c99b1f2d3ba9668be.

Comment (cooperq) [inline src/background.js]: Yes, removed.

**Comment by ghostwords** [comment]
Looks good to me.

**Pull request merged**